en/developer_guide/improve_vagrant_with_kvm.rst

```diff
@@ -0,0 +1,76 @@
+Improve Vagrant with KVM
+========================
+
+Context
+-------
+
+A simple installation of `vagrant`_ will be working with `virtualbox`_. In some cases, you
+will notice that `virtualbox`_ is not the faster solution on the market. `KVM`_ can be a good
+alternative for the virtualization on your own computer.
+
+Basic installation
+------------------
+
+KVM installation
+~~~~~~~~~~~~~~~~
+
+You need to install all the packages for the vagrant plugin, and to allow `KVM`_ to run.
+
+.. code-block:: bash
+
+    $ aptitude install libxslt-dev libxml2-dev libvirt-dev zlib1g-dev qemu-utils libvirt-dev qemu-kvm
+    $ gem install ruby-libvirt -v '0.6.0'
```

Comment:
You may have to explain that you have to do (install) something in order to use `gem` command.